react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,40 @@
+.overlay {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  position: relative;
```

Comment:
I don't really understand what all of these properties are doing here. Is there anything you can remove or simplify?